src/cli/main.ts

```diff
@@ -71,14 +99,19 @@ const initFlagsFile = async () => {
 };
 
 const run = async () => {
-  const { command, flagsToDisable, flagsToEnable } = parseArgs(process.argv);
+  const { command, flagsToDisable, flagsToEnable, skipIfEnv } = parseArgs(
+    process.argv
+  );
 
   if (command === "init") {
     await initFlagsFile();
     return;
   }
 
   if (command === "override") {
+    if (shouldSkip(skipIfEnv)) {
+      return;
+    }
```

Comment:
i think we put this at the top of the `run()` here b/c the way it's documented it's a global CLI option and not specific to one command